src/lib/src/component/navigation/navigation.component.ts

```diff
@@ -152,7 +164,7 @@ export class NavigationComponent implements OnInit, OnDestroy {
   }
 
   refresh(): void {
-    this.retrieveWorkspaceRoot();
+    this.retrieveWorkspaceRootVia({ rebuild: true });
```

Comment:
should we disallow requesting a refresh while a previous refresh hasn't completed, yet?